securedrop/source_app/info.py

```diff
@@ -18,8 +18,8 @@ def tor2web_warning() -> str:
     def recommend_tor_browser() -> str:
         return render_template("use-tor-browser.html")
 
-    @view.route('/journalist-key')
```

Comment:
Rather than removing this, it might be better to convert it to a permanent redirect, just on the off-chance it's being used/linked by instance owners or sources. Something like this:
```
diff --git a/securedrop/source_app/info.py b/securedrop/source_app/info.py
index df43164eb..4f1a83644 100644
--- a/securedrop/source_app/info.py
+++ b/securedrop/source_app/info.py
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 import flask
-from flask import Blueprint, render_template, send_file, current_app
+from flask import Blueprint, render_template, send_file, current_app, redirect, url_for
 
 from io import BytesIO  # noqa
 
@@ -18,6 +18,10 @@ def make_blueprint(config: SDConfig) -> Blueprint:
     def recommend_tor_browser() -> str:
         return render_template("use-tor-browser.html")
 
+    @view.route('/journalist-key')
+    def download_journalist_key() -> flask.Response:
+            return redirect(url_for('.download_public_key'), code=301)
+    
     @view.route('/public-key')
     def download_public_key() -> flask.Response:
         journalist_pubkey = current_app.crypto_util.gpg.export_keys(
```

Reply:
Makes total sense!